content/product-engineering/planning-process.md

```diff
@@ -70,39 +70,38 @@ There are a few ways we track plans and celebrate progress at Sourcegraph.
 
 The [GitHub project tracker](https://github.com/orgs/sourcegraph/projects/214) is **the source of truth** for key results in product and engineering. There are a few tabs.
 
-- The **Department KRs** tab tracks the top level product and engineering key results that the VPs and Directors review every week at the [VP team sync](#vp-team-sync).
-- The **Org KRs** tab tracks all the org level key results (excluding those that are tracked at the department level). VPs and Directors review this monthly, or as-needed when something is escalated by a Director.
+- The **Department KRs** tab tracks the top level product and engineering key results that the VPs and Directors review every week at the [VP team sync](#vp-team-sync). Updates are tracked and communicated in the [OKR slides](#okr-slides).
+- The **Org KRs** tab tracks all the org level key results. VPs and Directors review this monthly, or as-needed when something is escalated by a Director.
+
+  - For each KR, the assignee is responsible for posting an update to the issue at the end of each week. The update should include:
```

Comment:
What's the value of mandating this each week vs. asking people to make updates in this format when they are available? Is every KRs expected to generate wins, risks, and next steps during every week of the quarter? Is it that we are worried people will forget to give updates if we don't make them weekly and mandatory?

Reply:
As discussed in the sync today, going to remove this expectation from the docs. Thanks for the feedback!